src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -244,6 +246,47 @@ internal static class PeWriter
             return true;
         }
 
+        private static void FixupChecksum(ExtendedPEBuilder peBuilder, BlobBuilder peBlob)
+        {
+            // Checksum fixup, workaround for https://github.com/dotnet/corefx/issues/25829
```

Comment:
Do we have a Roslyn issue tracking the CoreFX issue? If there is only a CoreFX issue worried we will lose track of undoing the work around.